### PR TITLE
Add env arguments to hugo

### PR DESCRIPTION
**Pull request by Bas-Man**
Hi,

This will add support for passing arguments to the hugo command.

This addresses issue: https://github.com/victoriadrake/hugo-remote/issues/3

I have tested this using my own branch and was able to minify the generated files.

### CODE REVIEW

File: action.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ if [ -d "${DEST}" ]; then
 fi
 
 echo '🍳 Build site'
-hugo -d ${DEST}
+hugo ${HUGO_ARGS} -d ${DEST}
```
Comment (dentarg) [inline action.sh]: This script will fail on unset variables so I think you want this 

```suggestion
hugo ${HUGO_ARGS:-""} -d ${DEST}
```

This should use the empty string if `HUGO_ARGS` isn’t set (see https://stackoverflow.com/questions/12262696/using-unset-vs-setting-a-variable-to-empty#comment16445973_12263914)

**Comment by Bas-Man** [comment]
When setting HUGO_ARGS: with nothing
This does not appear to cause issues.
The string should never be empty would probably be the right course of action. But I will run some additional tests.

**Comment by Bas-Man** [comment]
Setting HUGO_ARG: '' 
caused no issue

File: action.sh (hunk 1 above)
Comment (Bas-Man) [reply action.sh]: This variable is always set. Well I think it is. as it is defined in the work flow. And should not be removed?

**Comment by Bas-Man** [comment]
I think this looks up now. I have tested in through the various cases.
the ${HUGO_ARGS:-""} deals with the case where a user may have removed the env: from the hugo step

I think this should never really be removed. But in the case that someone comments it out. The script will still run to completions.

Thanks @dentarg

File: README.md [hunk 2]
```diff
@@ -28,6 +28,14 @@ This action expects variables for:
 - `DEST`, the name of the destination directory you would like the site to build to,
 - `TOKEN`, in the form `${{ secrets.TOKEN }}` assuming `TOKEN` is the name you've used.
 
+You can optionally set a string value for the variable named `HUGO_ARGS`. Note these should be set on a single line.
+
+- `HUGO_ARGS` in the form 'string' Here are some examples:
+- `HUGO_ARGS: '--minify'`
+- `HUGO_ARGS: '-D -F'`
+  
+If you do not wish to pass any arguments. Please leave HUGO_ARGS blank as `HUGO_ARGS:`
```
Comment (dentarg) [inline README.md]: Now that the script does `${HUGO_ARGS:-""}`, the user shouldn't need to set `HUGO_ARGS` as blank, they should just be able to omit the whole `env:` thing

Comment (Bas-Man) [reply README.md]: First, Let me apologise for the rubbish english I have been writing lately. I have not been paying attention to the auto correct.
Second, You are absolutely correct. This is a good example of defensive programming and makes for a far more robust implementation.

Comment (victoriadrake) [reply README.md]: Merged in. Thank you both!

**Pull request closed without merging**